src/Firebase/Database/SseClient.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Kreait\Firebase\Database;
+
+use GuzzleHttp\Client;
+use Kreait\Firebase\Exception\ApiException;
+
+/**
+ * SSE Client to retrieve data from streaming rest API
+ */
+class SseClient
+{
+    const RETRY_DEFAULT_MS = 3000;
+    const END_OF_MESSAGE = "/\r\n\r\n|\n\n|\r\r/";
+
+    /**
+     * @var Client
+     */
+    private $apiClient;
+
+    /**
+     *
+     * @var GuzzleHttp\Psr7\Response
```

Comment:
I personally would prefer `Psr\Http\Message\ResponseInterface` here. Guzzle does return a `GuzzleHttp\Psr7\Response`, but if one day we change the ApiClient implementation, this saves us one line of PHPDoc that we have to change ^^.